build.gradle

```diff
@@ -41,6 +42,28 @@ distributions {
     }
 }
 
+jar {
+    manifest {
+        attributes ("Main-Class" : "io.logz.jmx2graphite.Jmx2GraphiteJolokia",
+                    "Premain-Class": "io.logz.jmx2graphite.Jmx2GraphiteJavaAgent")
+    }
+}
+
+task javaAgent(dependsOn: [shadowJar]) {
+}
+
+shadowJar {
+    manifest.from jar.manifest
+    classifier = 'javaagent'
+    relocate 'org.apache', 'io.logz.org.apache'
+    relocate 'com.fasterxml.jackson', 'io.logz.com.fasterxml.jackson'
+    relocate 'io.dropwizard.metrics', 'io.logz.ch.io.dropwizard.metrics'
+    relocate 'com.typesafe', 'io.logz.com.typesafe'
```

Comment:
I really want to move forward on this..
Lets open an issue and address that in the future?